components/tensorflow-notebook-image/Makefile

```diff
@@ -11,18 +11,34 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 
-all: container
-
 TAG?=$(shell git rev-parse HEAD)
-REGISTRY?=gcr.io/kubeflow
+GCR_REGISTRY?=gcr.io/kubeflow
+DOCKERHUB_REGISTRY?=hub.docker.com/r/kubeflow
```

Comment:
How about using the `PROJECT_ID` macros?

Reply:
I can do that. I noticed that each Makefile already was using different variables; I can standardize it to all use PROJECT_ID though